SARS-CoV-2_protocols/SARS-CoV-2-2000bp/data/run_configuration.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think this RAMPART repository should hold no custom configuration. All the custom protocols and run_configuration files should be located in nanocrop, where we have developed terminal user interface to let the user select from several protocols etc. In RAMPART should only be default examples of protocols and run configurations as they are in original repository. This fork should implement strictly RAMPART enhancements, such as new pipeline and internal modifications.

Reply:
okay, all custom configuration except the new pipeline itself is removed. However, I think that leaving the default ebola examaples in the repository might be misleading for now, because the pipeline itself contains a virus-specific file which describes mutations for the variants we are matching. I think we should then move the pipeline itself to custom protocols in nanocrop too.

Reply:
Is it fixed mechanism in RAMPART that default pipeline is loaded from default_protocol/ folder or can it be configured somewhere?

Reply:
it is possible to change it by changing one constant in RAMPART code (DEFAULT_PROTOCOL_PATH in server/config/getInitialConfig.js). It is not configurable form command line or config file, i guess.

Reply:
but you can add a new pipeline to RAMPART by creating a `pipelines.json` file in your custom protocol folder. so we can remove our strand-matching pipeline from the default protocol and add it to your config in nanocrop.

Reply:
Great, thak you! That's a way to go, I think. Firstly I agree that Ebola examples might be misleading when default pipeline is heavily focused on SARS-CoV-2. But I think that default pipeline should remain the original default pipeline, not our SARS-CoV-2 focused one. Original default pipeline is much more versatile, intuitive, well documented and can be used for any sequenced genome once a proper reference file is in particular used protocol. So I would place default pipeline to default_protocol as it was. I also don't think that default_protocol folder is well named. Because it contains default_protocol together with default pipeline. Now that we have more pipelines (and possibly more coming) I would create "pipelines/" folder in repository root a there folders "default_pipeline" and folder for our pipeline (you can come up with better name than me :) ). And default protocol would hold just default_protocol. pipelines.json file in "default_protocol/" would have modified path to point to newly placed default pipeline.
As you suggested, the right pipeline to be used for our experiment would be specified in nanocrop.

When it comes to example stuff, I would like to preserve them (default pipeline will support them). It just boosted my learning curve with RAMPART so much when I used it for the first time. And we have to support newcomers in my opinion. It doesn't matter that it's Ebola genome, it's just a few clicks and you can run RAMPART and see what it does, that's very useful content from original repository in my opinion. So I would preserve example fastq files and also I wouldn't move examples to "old/" folder because they seem outdated there. I would leave them without change. But I can be wrong. Feel free to share your thoughts @ppershing @janka000 .

Reply:
Having example fastq files was also very helping when I found out that fastq files generated by our compbio basecallers were not compatible with RAMPART. It was very easy to compare them against example file and identify missing metadata in our generated fastq.